bazel/repository_locations.bzl

```diff
@@ -309,7 +309,7 @@ REPOSITORY_LOCATIONS_SPEC = dict(
         project_desc = "Library to randomly mutate protobuffers",
         project_url = "https://github.com/google/libprotobuf-mutator",
         version = "1.1",
-        sha256 = "fd299fd72c5cf664259d9bd43a72cb74dc6a8b9604d107fe2d2e90885aeb7c16",
+        sha256 = "9b862857a725a07287dc2dd182f32b76b38b8129b38aad3e26ebceccd9d6c620",
```

Comment:
@caschoener this should not change if the version has not changed

Reply:
I'll change it back.  Was trying to fix a strange checksum error, thought maybe the sha56 was incorrect

Reply:
if you require updated deps for your PR - perhaps try and update these independently - that is generally preferable from review/revert pov

Reply:
After talking with JP apparently github tarball checksums aren't stable so this might be a good change if it makes tests pass

Reply:
(otherwise - you definitely do not want to change any hashes without changing the corresponding version)

Reply:
> After talking with JP apparently github tarball checksums aren't stable so this might be a good change if it makes tests pass

they are stable otherwise we would not be able to hash them

Reply:
You may want to check the envoy mobile collaboration slack channel for discussion on this point